tests/reg_tests/test_euler_tutorial_wing.py

```diff
@@ -0,0 +1,195 @@
+from __future__ import print_function
+import unittest
+import numpy
+from baseclasses import BaseRegTest
+import os
+import sys
+baseDir = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(baseDir,'../../'))
+from python.pyADflow import ADFLOW
+import copy
+from baseclasses import AeroProblem
+# from reg_tests.commonUtils import standard_test, adflowDefOpts, defaultFuncList
+from commonUtils import *    
+from runpy import run_path
+
+
+class TestSolve(unittest.TestCase):
+    '''
+    Test 9: MDO tutorial -- Euler -- Solution Test
+    '''
+    N_PROCS = 4
+
+    def setUp(self):
```

Comment:
You could put the solve into the setUpClass, but I think it should stay as a test so that if it fails it's clear it is a problem with the code being tested not the tests.

We could remove any files that were created by the tests in the `tearDown`. What do you think about that?

Reply:
Oh sorry I was confused -- I thought the solve was already in the `setUp` routine. Since `setUp` is being called every time it doesn't really change anything, and you're right that this is more explicit. Given how quick these are I think it's fine as it is.

I'm not sure that makes sense, as the `tearDown` routine is called regardless of whether the test passed or not, and it's possible that the output files are useful in debugging. I was actually thinking more in terms of destroying objects and releasing memory.

Reply:
I'm pretty sure the __del__ method of the ADFLOW is being called after each test since I'm encountering an error there on myfork of adflow... 

So, I don't think we need to destroy anything, although I'm not 100% sure.